scripts/requirements.txt

```diff
@@ -11,7 +11,7 @@ tensorflow==2.0.1; python_version > '2.7'
 tensorflow==2.0.0; python_version < '3.5'   # Last version to support Python 2.7
 UISoup==2.5.7
 pyobjc==5.2; sys_platform == 'darwin'
-llvmlite==0.31.0
+llvmlite==0.33.0
```

Comment:
I don't think this version support either Python 2.7 or 3.5. I recently ran into this issue myself (while trying to support Python 3.8). Here is the change I used:
```
llvmlite==0.33.0; python_version == '3.8'
llvmlite==0.31.0; python_version != '3.8'
```